src/Processors/AugmentTags.php

```diff
@@ -15,6 +15,22 @@
  */
 class AugmentTags implements ProcessorInterface
 {
+
+    /** @var array<string>  */
+    protected $unusedTagsToKeepWhitelist = [];
+
+    public function __construct(array $unusedTagsToKeepWhitelist = [])
+    {
+        $this->unusedTagsToKeepWhitelist = $unusedTagsToKeepWhitelist;
+    }
+
+    public function setUnusedTagsToKeepWhitelist(array $unusedTagsToKeepWhitelist): AugmentTags
```

Comment:
```suggestion
    /**
     * Whitelist tags to keep even if not used. <code>*</code> may be used to keep all unused.
     */
    public function setUnusedTagsToKeepWhitelist(array $unusedTagsToKeepWhitelist): AugmentTags
```

The docblock on the `setXXX()` method is used to auto-generate the processor reference, so adding something here would be good.